docs/paywall3.md

```diff
@@ -683,9 +683,28 @@ paywall.on('preview', function(e) {
 });
 ```
 
-## Standalone Paywall Functions
+## Standalone Functionalities
 
-Paywall 3.0 also supports standalone functionalities. Apart from the login/logout button functionality, accessing 'My Account' section, and invoking the 'Change Password' screen, supported in the previous version, Paywall 3.0 introduces and adds to the list the register button functionality, the setup of a language, and accessing the 'Credit Card Details' screen.
+Much like the previous version, Paywall 3.0 also supports the standalone functionalities involving the login/logout button functionality, the 'Change Password' screen, the card management screen, as well as accessing the 'My Account' section of the paywall application.
+
+Although you may use the paywall's JavaScript methods to invoke some of the functionalities or certain paywall screens, you can achieve the same functionalities by adding HTML classes to the elements on the page. 
+
+Here is the list of the HTML classes that can be used for creating each of the standalone functionalities: 
+
+| HTML class	|   Description |
+|--------------|---------------|
+| `inplayer-paywall-login`	| Invokes the login screen of the paywall |
+| `inplayer-paywall-logout`	| Invokes the logout action  |
+| `inplayer-paywall-account`	 | Invokes the 'My Account' screen of the registered user |
+| `inplayer-paywall--ccm`      | Invokes the  default card management screen |
```

Comment:
i think one `-` should be removed. instead it should be inplayer-paywall-ccm ; can you please also change this in the paywall v2 guide?